src/constants.ts

```diff
@@ -130,6 +130,36 @@ export const GLOWING_CRYSTAL_IDS = [
   7568,
 ];
 
+/**
+ * IDs of the Ice demon from the Chambers of Xeric.
+ */
+export const ICE_DEMON_IDS = [
+  7584, // reg
+  7585, // cm
+];
+
+/**
+ * IDs of the Fragment of Seren.
+ */
+export const FRAGMENT_OF_SEREN_IDS = [
+  8917, 8918, 8919, 8920
+];
+
+/**
+ * IDs of monsters that calculate their magical defence using the defence stat.
+ * https://twitter.com/JagexAsh/status/1689566945635438592
+ **/
+export const ICE_DEMON_STYLE_DEF_IDS = [
+  ...ICE_DEMON_IDS,
+  ...VERZIK_P1_IDS,
+  10833, 10834, 10835, // verzik entry mode
+  8372, 8373, 8374, // verzik normal mode
+  10850, 10851, 10852, // verzik hard mode
```

Comment:
Merge these into a VERZIK_IDS constant which can be reused here and in the party size list